### PR TITLE
feat: keychain api, to replace the one from the wallet feature

### PR DESCRIPTION
**Pull request by mvayngrib**
## Summary

this exports the `exodus.keychain.exportKey` SDK method, meant to replace `exodus.wallet.{getPrivateKey,getPublicKey}`

closes https://github.com/ExodusMovement/exodus-hydra/issues/5749

### CODE REVIEW

File: features/keychain/api/__tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+const otherSeed = mnemonicToSeed(otherMnemonic)
+const seedId = getSeedId(seed)
+const otherSeedId = getSeedId(otherSeed)
+const solanaKeyId = new KeyIdentifier({
+  assetName: 'solana',
+  derivationAlgorithm: 'BIP32',
+  derivationPath: "m/44'/501'/0'/0/0",
+})
+
+// gen fixtures
```
Comment (mvayngrib) [inline features/keychain/api/__tests__/index.test.js:21]: leaving this so it's easy to add more if needed, but lmk if u want this gone

File: features/keychain/index.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import keychainDefinition from './module/keychain'
+import moduleDefinition from './module/keychain'
+import apiDefinition from './api'
 
 const keychain = ({ cachePublicKeys }) => {
   // TODO: support caching
   return {
     id: 'keychain',
-    definitions: [keychainDefinition],
+    definitions: [moduleDefinition, apiDefinition],
```
Comment (diegomura) [inline features/keychain/index.js:8]: Shouldn't this be 
```suggestion
    definitions: [{ definition: moduleDefinition }, { definition: apiDefinition }],
```

Comment (mvayngrib) [reply features/keychain/index.js:8]: ha, how did this work?

**Review by sparten11740 — DISMISSED** [review]
utACK

**Review by sparten11740 — APPROVED** [review]

**Review by diegomura — APPROVED** [review]
utACK

**Pull request merged**